src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/readme.md

```diff
@@ -4,95 +4,364 @@ Reference documentation for the current version of the REST API supported by the
+|--------|-------------------------------------------------------|
+| POST   | {endpoint}/softwareEntitlements?api-version={version} |
 
-Sample: `https://samples.westus.batch.azure.com/softwareEntitlements/?api-version=2017-99-99-9.9`
+| Placeholder | Type   | Description                                                                                                                                                                                                                                                                                           |
+|-------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| endpoint    | string | The Batch account URL endpoint supplied by Azure Batch via the environment variable AZ_BATCH_ACCOUNT_URL. <br/>This may have a trailing `/` ("https://demo.westus.batch.azure.com/") or it might not ("https://demo.westus.batch.azure.com"); clients should be prepared to handle either case correctly. |
+| version     | string | The API version of the request.                                                                                                                                                                                                                                                                       |
 
-| Placeholder | Type   | Description                                                                                                                                                                                                                                                                                                          |
-|-------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| endpoint    | string | The Batch account URL endpoint supplied by Azure Batch via the environment variable `AZ_BATCH_ACCOUNT_URL`.<br/>This may have a trailing `/` (e.g. `https://demo.westus.batch.azure.com/`) or it might not (e.g. `https://demo.westus.batch.azure.com`); clients should be prepared to handle either case correctly. |
-| version     | string | The API version of the request. <br/> Specify version `2017-99-99.9.9` or higher. <br/> For older API versions, see the end of this document. <br/> All Batch API versions are listed @ https://docs.microsoft.com/en-us/rest/api/batchservice/batch-service-rest-api-versioning                                     |
-
-The following shows a sample JSON payload for the request:
+The payload for the request is a JSON document:
```

Comment:
Clarify: do we require an `Accept` header or will it assume JSON by default?

Reply:
Assumes JSON by default.